src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/DownstreamBuildAction.java

```diff
@@ -0,0 +1,53 @@
+package org.jenkinsci.plugins.workflow.support.steps.build;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.model.InvisibleAction;
+import hudson.model.Item;
+import hudson.model.ItemGroup;
+import hudson.model.Run;
+import org.springframework.security.access.AccessDeniedException;
+
+/**
+ * Tracks the downstream build triggered by the {@code build} step.
+ * <p>Note that {@link #getBuildNumber} may return null if the build is in the queue or gets cancelled from the queue.
```

Comment:
Better moved to Javadoc of that method.

Reply:
https://github.com/jenkinsci/pipeline-build-step-plugin/pull/127/commits/1a96a2b25711b9f2b6cde3632747747e67faaaf4